pkg/abstractions/endpoint/endpoint.go

```diff
@@ -2,12 +2,13 @@ package endpoint
 
 import (
 	"context"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"net/http"
 	"time"
 
+	jsoniter "github.com/json-iterator/go"
```

Comment:
Why can't we name this `json` instead of `jsoniter` instead? Looks like it worked fine over here https://github.com/beam-cloud/beta9/pull/816/files#diff-3094a48851e225b16303320d7a7a55a493ecd14c499dec5418585eff10d1debbR17

Reply:
See comment in slack